application/frontend/controllers/MfaController.php

```diff
@@ -90,7 +91,7 @@ public function actionVerify(int $id)
 
         // Strip spaces from $value if string
```

Comment:
Let's also update this comment to clarify the purpose of this `is_string()` check (i.e. to see whether is TOTP, and thus a string of digits, or not, and thus could be anything and shouldn't have non-digits deleted).